rule_schema_v2.atd

```diff
@@ -352,6 +352,8 @@ type formula = {
 
   (* NEW: since 1.74 *)
   ?fix: string option;
+
+  ?when_: string option;
```

Comment:
would also be nice to add a `(* NEW: since ... *)`!

you can just find the most recent version of Semgrep here https://github.com/semgrep/semgrep and increment that